workitem/field_definition.go

```diff
@@ -82,8 +89,19 @@ func (f FieldDefinition) Equal(u convert.Equaler) bool {
 
 // ConvertToModel converts a field value for use in the persistence layer
 func (f FieldDefinition) ConvertToModel(name string, value interface{}) (interface{}, error) {
-	if f.Required && (value == nil || (f.Type.GetKind() == KindString && strings.TrimSpace(value.(string)) == "")) {
-		return nil, fmt.Errorf("Value %s is required", name)
+	if f.Required {
+		if value == nil {
+			return nil, fmt.Errorf("Value %s must not be nil", name)
+		}
+		if f.Type.GetKind() == KindString {
+			sVal, ok := value.(string)
+			if !ok {
+				return nil, errs.Errorf("failed to convert '%+v' to string", spew.Sdump(value))
+			}
+			if strings.TrimSpace(sVal) == "" {
+				return nil, errs.Errorf("Value '%s' must not be empty: %+v", name, spew.Sdump(value))
```

Comment:
I don't think you need to use `spew.Sdump()` here, since we know the value is empty.

Reply:
Right. Fixed in b0f9174b.